code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -89,8 +90,11 @@
 			emote("me", EMOTE_AUDIBLE, "snuffles")
 
 	if(nutrition >= MOUSETFAT)
-		visible_message("<span class = 'warning'>\The [src] explodes!</span>")
```

Comment:
instead of creating a var for this (less flexible, var bloat), instead use an override (more maintainable, more oop, more flexible)

at L93, call a new proc, e.g.: `mouse_overeat()`, which you can override for the transmog subtype

```
/mob/living/simple_animal/mouse/proc/mouse_overeat()
  gib()

/mob/living/simple_animal/mouse/transmog/mouse_overeat()
  transmog_death()
```

Reply:
will try to implement